tests/test_data_slices.py

```diff
@@ -0,0 +1,36 @@
+import sys
+import os
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../src')))
+
+import numpy as np
+from sklearn.model_selection import train_test_split
+from src.define_train_model import train_model, define_params, define_model, load_data
+
+
+def test_model_on_data_slices():
```

Comment:
Nice work! Could you indicate in comments on top that this test belongs under the category "Model Development" to make it clear for the graders we covered it? :smile: